mathjax3-ts/input/tex/ams_cd/AmsCdMethods.ts

```diff
@@ -122,14 +123,14 @@ AmsCdMethods.arrow = function(parser: TexParser, name: string) {
         if (a) {
           let nodeA = new TexParser(a, parser.stack.env, parser.configuration).mml();
           let mpadded = parser.create('node', 'mpadded', [nodeA], pad);
-          NodeUtil.setAttribute(mpadded, 'voffset', '1em');
+          NodeUtil.setAttribute(mpadded, 'voffset', '.1em');
```

Comment:
Good catch. That's the value of MJ2.

Reply:
It must have gotten edited somehow, because its not the value in the `extensions/AMScd.js` file.  Perhaps a global edit gone wrong.  No matter, we spotted it.